### PR TITLE
build/deploy: Update QEMU to speed up emulated builds 

### PR DESCRIPTION
**Pull request by klutchell**
QEMU v5 has quite a few improvements over v4, and the speed
difference when emulating arm is quite noticible.

We tested this with, and without, our single-core limitation
patch and have not been able to reproduce the stability
issues we were seeing in v4 so the patch was removed in
this release.

Change-type: patch
Connects-to: balena-io/balena-io#2340
Signed-off-by: Kyle Harding <kyle@balena.io>

---
Please check the CONTRIBUTING.md file for relevant information and some
guidance. Keep in mind that the CLI is a cross-platform application that runs
on Windows, macOS and Linux. Tests will be automatically run by balena CI on
all three operating systems, but this will only help if you have added test
code that exercises the modified or added feature code.

Note that each commit message (currently only the first line) will be
automatically copied to the CHANGELOG.md file, so try writing it in a way
that describes the feature or fix for CLI users.

If there isn't a linked issue or if the linked issue doesn't quite match the
PR, please add a PR description to explain its purpose or the features that it
implements. Adding PR comments to blocks of code that aren't self explanatory
usually helps with the review process.

If the PR introduces security considerations or affects the development, build
or release process, please be sure to highlight this in the PR description.

Thank you very much for your contribution!

### CODE REVIEW

**Review by pdcastro — COMMENTED** [review]
Great job @klutchell !! 💯 

Just being picky about the Changelog, which end users will be looking at. Only the first line of the commit gets copied to the Changelog, so at moment it would look like:

* qemu: update to v5.2.0+balena1

But many CLI users may not know or remember what `qemu` is about, and won't know which CLI commands are affected or the benefit of this CLI release. I suggest rewording the first line of the commit to:

`build/deploy: Update QEMU to speed up emulated builds`

So the Changelog reads:

* build/deploy: Update QEMU to speed up emulated builds

Then CLI users of `balena build -e` and `balena deploy --build -e` will realise that their builds are affected and should run faster. And in case of build problems, users may be able to associate it with this CLI release, to help with any investigations.

**Comment by klutchell** [comment]
@pdcastro Good to know! I didn't know that the changelog used the first line of the commit for this repo, I'm used to adding `Changelog-entry:` to the bottom of my commit template. I guess this is different for each repo? Does balena lean one way or the other regarding how the changelog picks things up?

**Comment by pdcastro** [comment]
> @pdcastro Good to know! I didn't know that the changelog used the first line of the commit for this repo, I'm used to adding `Changelog-entry:` to the bottom of my commit template. I guess this is different for each repo? Does balena lean one way or the other regarding how the changelog picks things up?

This repo should be treated the same as any other, and the default (in the absence of `Changelog-entry:`) is to copy the first line only. I had not noticed - my bad - that your previous commit had a `Changelog-entry` footer:

```
Changelog-entry: update qemu to v5.2.0 with balena patches [klutchell]
```

But the same review comment would have applied. :-)

**Review by pdcastro — APPROVED** [review]
